docs/.vuepress/config.js

```diff
@@ -183,7 +182,10 @@ module.exports = {
                 title: 'Further reading',
                 sidebarDepth: 1,
                 collapsable: false,
-                children: ['/concepts/further-reading/academic-papers']
+                children: [
+                  '/concepts/usage-ideas-examples',
```

Comment:
Suggest actually moving this into the "IPFS 101" section of the menu, maybe before "Glossary". It's such a useful piece of content that I'm worried it'll get lost down here.

Reply:
That's a fair point. I'll move it up. Thanks!